src/Compilers/Core/Portable/Operations/ArgumentKind.cs

```diff
@@ -21,14 +21,20 @@ public enum ArgumentKind
 
         /// <summary>
         /// Argument is a param array created by compilers for the matching C# params or VB ParamArray parameter. 
-        /// Note, the value is a an array creation expression that encapsulates all the elements, if any.
+        /// Note, the value is an array creation expression that encapsulates all the elements, if any.
         /// </summary>
-        ParamArray = 0x2, // PROTOTYPE(ParamsCollections): Reuse or add a special kind for non-array params collections?
+        ParamArray = 0x2,
 
         /// <summary>
         /// Argument is a default value supplied automatically by the compilers.
         /// </summary>
-        DefaultValue = 0x3
+        DefaultValue = 0x3,
+
+        /// <summary>
+        /// Argument is a param collection created by compilers for the matching C# params y parameter. 
```

Comment:
Not sure what the `y parameter` at the end here is?

Reply:
> Not sure what the `y parameter` at the end here is?

I guess the remainder artifact of "ParamArray"